src/scripting/game_lua_kernel.cpp

```diff
@@ -4272,6 +4261,37 @@ game_lua_kernel::game_lua_kernel(game_state & gs, play_controller & pc, reports
 	luaL_setfuncs(L, audio_callbacks, 0);
 	lua_setfield(L, -2, "audio");
 	lua_pop(L, 1);
+	
+	// Create the paths module
+	cmd_log_ << "Adding paths module...\n";
+	static luaL_Reg const path_callbacks[] {
+		{ "find_cost_map",             &dispatch<&game_lua_kernel::intf_find_cost_map              >        },
+		{ "find_path",                 &dispatch<&game_lua_kernel::intf_find_path                  >        },
+		{ "find_reach",                &dispatch<&game_lua_kernel::intf_find_reach                 >        },
+		{ "find_vacant_tile",          &dispatch<&game_lua_kernel::intf_find_vacant_tile           >        },
+		{ "find_vision_range",         &dispatch<&game_lua_kernel::intf_find_vision_range          >        },
+		{ nullptr, nullptr }
+	};
+	lua_getglobal(L, "wesnoth");
+	lua_newtable(L);
+	luaL_setfuncs(L, path_callbacks, 0);
+	lua_setfield(L, -2, "paths");
+	lua_pop(L, 1);
+	
+	// Create the sync module
+	cmd_log_ << "Adding sync module...\n";
+	static luaL_Reg const sync_callbacks[] {
+		{ "invoke_command",    &intf_invoke_synced_command },
+		{ "run_unsynced",      &intf_do_unsynced },
+		{ "evaluate",          &intf_synchronize_choice },
+		{ "multi_evaluate",    &intf_synchronize_choices },
```

Comment:
I'd prefer `multi_evaluate` to be `evaluate_multiple`.

Reply:
That's funny, because that was my first thought, but then I changed it to this.